WcaOnRails/config/locales/carrierwave.en.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  errors:
+    messages:
+      carrierwave_processing_error: failed to be processed
+      carrierwave_integrity_error: is not of an allowed file type
+      carrierwave_download_error: could not be downloaded
```

Comment:
Same thing as for kaminari: I took it from the locales carrierwave supplies :)
I fixed these.